components/ClaimableBooksSection.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <aside class="bg-primary/5">
+    <UCollapsible
+      v-if="claimableNFTClassIdsCount > 0"
+      :default-open="true"
+      :ui="{ content: 'border-b border-accented' }"
+    >
+      <header
+        :class="[
+          'group',
+          'bg-primary hover:bg-primary/75',
+          'cursor-pointer',
+          'transition-colors',
+        ]"
+      >
+        <div
+          :class="[
+            'flex',
```

Comment:
<img width="541" height="62" alt="截圖 2025-09-18 09 59 16" src="https://github.com/user-attachments/assets/22c5b861-3978-4a0b-8dd6-ba75ddbd958d" />

```suggestion
            'flex',
            'items-center',
```